packages/matrix/tests/login.spec.ts

```diff
@@ -73,6 +79,30 @@ test.describe('Login', () => {
     await expect(page.locator('[data-test-login-form]')).toBeVisible();
   });
 
+  test('the profile reflects display name changes', async ({
+    page,
+    synapse,
+  }) => {
+    page.on('console', (msg) => console.log(msg.text()));
+    await login(page, 'user1', 'pass');
+
+    await page.locator('[data-test-profile-icon-button]').click();
+    await expect(page.locator('[data-test-profile-display-name]')).toHaveText(
+      'user1',
+    );
+
+    await updateDisplayName(
+      synapse,
+      registeredUser!.userId,
+      registeredUser!.accessToken,
+      'newname',
+    );
+
+    await expect(page.locator('[data-test-profile-display-name]')).toHaveText(
+      'newname',
+    );
+  });
```

Comment:
This test is failing but you can see it working here locally:

![screencast 2023-12-14 16-52-17](https://github.com/cardstack/boxel/assets/43280/11790fc1-20b6-4828-97f5-ab1d99f2cbce)

You can also see that in the test [the call to change the name appears to succeed](https://github.com/cardstack/boxel/actions/runs/7225512559/job/19689119521?pr=909#step:8:194).